.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
### Highlight 

- Used correct  **GitHub** flow  
- Professional  comment with polite and friendly behavior